src/08-tasks/tasks.apib

```diff
@@ -42,6 +42,12 @@ Get a list of tasks.
                 + work_type (object, nullable)
                     + type: `workType` (string)
                     + id: `32665afd-1818-0ed3-9e18-a603a3a21b95` (string)
+                + responsible (object)
```

Comment:
tbh, now that I see this, why don't call it "assigned to" like in the UI?

Reply:
> tbh, now that I see this, why don't call it "assigned to" like in the UI?

Before this we had assignee id, but we checked and in the rest of the app the "term" assignee was not used anywhere, they used "responsible user", so to keep everything with the same naming, since the meaning is the same (the only difference was the naming), we decided to change to responsible_user. After that we decided that the responsible_user would not work, because it could be a user or a team so we created a VO for Responsible with a type and id, so we could easily manage the user and team type. If we change this to "assigned to" like the UI, we will introduce a new term on the API and it will be used only, because the UI has it as "assigned to" while the rest of the app uses the term "responsible user" for this type of situation.

Reply:
If "assigned to" makes more sense we can also change the UI.

Reply:
My UI shows assigned to by the way:

![image](https://user-images.githubusercontent.com/194377/53339095-9f323c00-3905-11e9-90d7-6d6d63a13667.png)

Reply:
I'm convinced we should change this to `assignedTo`. `assignee` was weird, but `assignedTo` makes sense to me.